src/app/fyle/split-expense/split-expense.page.ts

```
@@ -1115,4 +1113,41 @@
+    const { data } = await reviewModal.onWillDismiss();
+
+    if (data) {
+      if (data.action === 'close') {
+        if (this.reportId) {
+          this.router.navigate(['/', 'enterprise', 'my_view_report', { id: this.reportId }]);
+        } else {
+          this.router.navigate(['/', 'enterprise', 'my_expenses']);
+        }
+      } else if (data.action === 'navigate') {
+        const expense = data.expense;
+        let category = expense?.category?.name?.toLowerCase();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'category' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)